the-blue-alliance-ios/Frameworks/MyTBAKit/Models/MyTBARegister.swift

```diff
@@ -12,7 +12,7 @@ struct MyTBARegisterRequest: Codable {
 
 struct MyTBARegisterResponse: MyTBAResponse, Codable {
 
-    var code: String
+    var code: String // TODO: File a bug with web - this code needs to be an Int
```

Comment:
See if we can track this down too

Reply:
This is for sure some bug on our end - iOS side, or integration side, or something. It's not an issue with web.